covid.py

```diff
@@ -1 +1,67 @@
-#this is comment
+import argparse
+import json
+import requests
+
+
+def parser():
```

Comment:
`def parser():` suggest you'll return, well, `parser`. In fact, you're returning parsed arguments. What do you think about renaming to `get_parsed_arguments()`?